docs/components/modeler/bpmn/compensation-events/compensation-events.md

```diff
@@ -0,0 +1,76 @@
+---
+id: compensation-events
+title: "Compensation events"
+description: "Compensation events are used to undo tasks that have already been executed."
+---
+
+Compensation events assist with undoing steps that were already successfully completed in the case that their results are no longer desired and need to be reversed.
+
+![Process with compensation throw event](assets/compensation-throw-event.gif)
+
+The example above shows the execution of a compensation event:
+
+1. After the service task `A` is completed, the process reaches the compensation intermediate throw event.
+2. This invokes the compensation handler `undo A` attached to the compensation boundary event.
+3. Once the compensation handler `undo A` is completed, the process completes the compensation intermediate throw event and takes the outgoing sequence flow.
+
+:::info
+**Compensation activity:** An activity with a compensation boundary event attached. This can be compensated only if completed first.  
+**Compensation handler:** An activity with a compensation marker, and the activity in charge of reverting the effects of the compensation activity attached. For more information, visit the [compensation handler documentation](../compensation-handler/compensation-handler.md).
+:::
+
+## Triggering compensation
+
+By default, a compensation intermediate throw or end event triggers the compensation within its scope. If the compensation throw event is on the process level, it invokes all at once all the compensation handlers of the process without any specific order. The compensation throw event remains active until all the compensation handlers are completed.
+
+Compensation handlers are triggered for [subprocesses](#embedded-subprocess) but not for child processes.
+
+## Triggering compensation for an activity
+
+Besides the broadcasting of the compensation event, a compensation throw event can trigger the compensation for a specific activity. Triggering the compensation for a specific activity can be used to enforce that compensation handlers are invoked synchronously in a given order.
+
+![Trigger compensation for a give activity](assets/compensation-activity-ref.png)
+
+On a compensation intermediate throw or end event, it is possible to specify the activity to compensate by using the property `activityRef`. The referenced activity must have a compensation boundary event and must be in the same scope of the compensation throw event.
+
+### Multi-instance activity
+
+If an activity is a multi-instance activity or multi-instance subprocess, the compensation handler is invoked once. The compensation handler is responsible for compensating the effect of all instances of the multi-instance activity.
```

Comment:
:wrench: We could link to "Multi-instance activity as compensation handler" as an alternative to a single compensation handler that undoes the effect of all instances.